lib/commons/aria/validate-attr-value.js

```diff
@@ -44,11 +47,15 @@ function validateAttrValue(node, attr) {
       }, list.length !== 0);
 
     case 'idref':
-      return !!(value && doc.getElementById(value));
+      try {
+        const doc = getRootNode(vNode.actualNode);
+        return !!(value && doc.getElementById(value));
+      } catch (e) {
+        throw new TypeError('Cannot resolve id references for non-DOM nodes');
```

Comment:
This can happen with unmounted nodes too. This seems like a more appropriate message:

```suggestion
        throw new TypeError('Cannot resolve id references for partial DOM');
```